mantidproject/fitbenchmarking
# All code depending on mantid now contained in mantid.py file

#### Description of Work

Fixes #5 
Laid the base for extending to other fitting algorithms/minimizers.

#### Testing Instructions

1. In the `mantidpython` terminal, run `example_runScripts.py`.
2. Check if the results are the same as before
